tests/functional/lib/helpers.js

```diff
@@ -182,6 +182,9 @@ const testElementExists = thenify(function (selector) {
  */
 const click = thenify(function (selector, readySelector) {
   return this.parent
+    .execute(() => {
+      window.scrollTo(0,0);
```

Comment:
@mozilla/fxa-devs Thoughts on this approach? Testing locally on small screens it works as intended. I originally tried using the `moveMouseTo` helper but that failed with same `obscures it`.

Reply:
This seems like it could scroll the item you are trying to click out of view if the element is far down the page, e.g., the Delete Account submit button. Webdriver explodes if trying to click an element that is not within the viewport.

Perhaps the way to approach this is to do the click then immediately capture any errors. If the error is about the element being obscured, do the scroll and re-click.

```js
.click()
.then(null, (err) => {
  if (/obscures it/.test(err.message)) {
     return this.parent
        .execute(() => {
           window.scrollTo(0,0);
        })
        .then(click(selector));
  } 
  // re-throw other errors
  throw err;
})
.findByCssSelector(selector)
...
```

Reply:
Thinking about this more, the above approach has a problem if the element is still obscured after scrolling, the test would end up in an infinite loop.

Maybe it should be something like:
```js
.click()
.then(null, (err) => {
  if (/obscures it/.test(err.message)) {
     return this.parent
        .execute(() => {
           window.scrollTo(0,0);
        })
        // I think this.parent should already be the element to click
        .click();
  } 
  // re-throw other errors
  throw err;
})
.end()
...
```